2018-12-13 08:03:45,042 - __main__ - INFO - Started the test_implementing
2018-12-13 08:03:45,042 - __main__ - INFO - Configuration file: logging.cnf
2018-12-13 08:03:45,042 - __main__ - INFO - ============================================================================
2018-12-13 08:03:45,042 - __main__ - INFO - Dataset to process: BSDL
2018-12-13 08:03:45,042 - __main__ - INFO - Data files are stored in: /home/petr/Projects/Valeo/RCW/data/Data_2017_02/
2018-12-13 08:03:45,042 - __main__ - INFO - Data for the scenario are in:
2018-12-13 08:03:45,042 - __main__ - INFO - 	 	 left_radar: CVUT_WORK_BSDL_left_t35_LeftRadar_RADAR
2018-12-13 08:03:45,042 - __main__ - INFO - 	 	 right_radar: None
2018-12-13 08:03:45,042 - __main__ - INFO - 	 	 left_dgps: CVUT_WORK_BSDL_left_t35_DGPS
2018-12-13 08:03:45,042 - __main__ - INFO - 	 	 right_dgps CVUT_WORK_BSDL_right_t45_DGPS:
2018-12-13 08:03:45,042 - __main__ - INFO - 	 	 both_dgps: 
2018-12-13 08:03:45,042 - __main__ - INFO - 	 	 logger_cfg_file: logging.cnf
2018-12-13 08:03:45,042 - __main__ - INFO - Radar to process: L
2018-12-13 08:03:45,042 - __main__ - INFO - ============================================================================
2018-12-13 08:03:45,043 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: A new track manager will be created with a gate:
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: 	 	 <data_containers.Gate object at 0x7f3ca2e87400>
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: 	 	 tracker_type {'filter_type': 'kalman_filter', 'dim_x': 4, 'dim_z': 2},
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: 	 	 Tsampl 0.05, number of tracks [0]
2018-12-13 08:03:45,043 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:03:45,043 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:45,043 - data_containers - DEBUG - UnAssignedDetectionList.__init__: list initialized, gate: dim_x=0.5, dim_y=0.3
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: 	 just created, number of unassigned dets 0
2018-12-13 08:03:45,043 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: A new track manager will be created with a gate:
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: 	 	 <data_containers.Gate object at 0x7f3c8a96be48>
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: 	 	 tracker_type {'filter_type': 'kalman_filter', 'dim_x': 4, 'dim_z': 2},
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: 	 	 Tsampl 0.05, number of tracks [0]
2018-12-13 08:03:45,043 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:03:45,043 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:45,043 - data_containers - DEBUG - UnAssignedDetectionList.__init__: list initialized, gate: dim_x=0.5, dim_y=0.3
2018-12-13 08:03:45,043 - track_management - DEBUG - __init__: 	 just created, number of unassigned dets 0
2018-12-13 08:03:45,043 - __main__ - INFO - Beams 0 will be processed: 1 times
2018-12-13 08:03:45,043 - __main__ - INFO - Beams 1 will be processed: 1 times
2018-12-13 08:03:45,044 - __main__ - INFO - Beams 2 will be processed: 1 times
2018-12-13 08:03:45,044 - __main__ - INFO - Beams 3 will be processed: 1 times
2018-12-13 08:03:45,044 - __main__ - INFO - ============================================================================
2018-12-13 08:03:45,044 - __main__ - INFO - Left Radar Data:
2018-12-13 08:03:45,044 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:03:45,093 - data_containers - DEBUG - DetectionList.append_data_from_m_file: points appended = 4189
2018-12-13 08:03:45,094 - __main__ - INFO - MCC Left start: 77073, end: 80262, dMCC=3189, number of detections: 4188.
2018-12-13 08:03:45,094 - __main__ - INFO - ============================================================================
2018-12-13 08:03:45,094 - __main__ - INFO - Data Preprocessor Settings:
2018-12-13 08:03:45,094 - __main__ - DEBUG - Just left radar will be processed 
2018-12-13 08:03:45,094 - __main__ - DEBUG - Number of processed MCCs from cnf file: 10000.
2018-12-13 08:03:45,094 - __main__ - INFO - Processing will start at: 77073, end: 80262, dMCC=3189.
2018-12-13 08:03:45,094 - __main__ - DEBUG - Inside of the MCC interval from 77073 to 80262: 
2018-12-13 08:03:45,094 - __main__ - DEBUG - ===========================================================================
2018-12-13 08:03:45,094 - __main__ - DEBUG - Filtering loop:
2018-12-13 08:03:45,094 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:03:45,095 - data_containers - DEBUG - DetectionList.get_lst_detections_selected: number of detections selected is 4188 MCCs from 77073 to 77073
2018-12-13 08:03:45,095 - __main__ - DEBUG - Filtering loop: Number of detections for a LR mcc 77073 is 1
2018-12-13 08:03:45,095 - track_management - DEBUG - new_detections: Tested will be new 1 detections
2018-12-13 08:03:45,095 - track_management - DEBUG - new_detections: 	 	 with MCCs from 77073 to 77073
2018-12-13 08:03:45,095 - track_management - DEBUG - new_detections: In a _lst_not_assigned_detections is 0 detections.
2018-12-13 08:03:45,095 - track_management - DEBUG - new_detections: 	 after 10 mccs removal: 0 detections.
2018-12-13 08:03:45,095 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77073
2018-12-13 08:03:45,095 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77073
2018-12-13 08:03:45,095 - data_containers - DEBUG - 	 at x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:45,095 - data_containers - INFO - UnAssignedDetectionList.new_detection:Detection stored in an unassigned list. Now it contains: 1 dets
2018-12-13 08:03:51,393 - __main__ - DEBUG - Predict cycle for each track in a list of 0 started for LR at mcc: 77073.
2018-12-13 08:03:51,393 - __main__ - DEBUG - New_track not ported/created
2018-12-13 08:03:51,394 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:03:51,395 - data_containers - DEBUG - DetectionList.get_lst_detections_selected: number of detections selected is 4188 MCCs from 77074 to 77074
2018-12-13 08:03:51,396 - __main__ - DEBUG - Filtering loop: Number of detections for a LR mcc 77074 is 2
2018-12-13 08:03:51,396 - track_management - DEBUG - new_detections: Tested will be new 2 detections
2018-12-13 08:03:51,396 - track_management - DEBUG - new_detections: 	 	 with MCCs from 77074 to 77074
2018-12-13 08:03:51,396 - track_management - DEBUG - new_detections: In a _lst_not_assigned_detections is 1 detections.
2018-12-13 08:03:51,396 - track_management - DEBUG - new_detections: 	 after 10 mccs removal: 1 detections.
2018-12-13 08:03:51,396 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77074
2018-12-13 08:03:51,396 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77074
2018-12-13 08:03:51,396 - data_containers - DEBUG - 	 at x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:51,397 - data_containers - INFO - UnAssignedDetectionList.new_detection:Detection stored in an unassigned list. Now it contains: 2 dets
2018-12-13 08:03:52,223 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77074
2018-12-13 08:03:52,224 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77074
2018-12-13 08:03:52,224 - data_containers - DEBUG - 	 at x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:52,224 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 2 
2018-12-13 08:03:52,224 - data_containers - DEBUG - 				a permutation of det1 77073, det2 77074
2018-12-13 08:03:52,224 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:52,224 - data_containers - DEBUG - 					 det2 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:52,226 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:52,226 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:52,226 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:52,226 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:52,226 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:52,226 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:52,226 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 2 
2018-12-13 08:03:52,226 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77073
2018-12-13 08:03:52,227 - data_containers - DEBUG - 					 det1 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:52,227 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:52,227 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:52,227 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:52,227 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:52,227 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:52,227 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:52,227 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:52,227 - data_containers - INFO - UnAssignedDetectionList.new_detection: Detection stored in an unassigned list. Now it contains: 3 dets
2018-12-13 08:03:52,844 - __main__ - DEBUG - Predict cycle for each track in a list of 0 started for LR at mcc: 77074.
2018-12-13 08:03:52,844 - __main__ - DEBUG - New_track not ported/created
2018-12-13 08:03:52,844 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:03:52,847 - data_containers - DEBUG - DetectionList.get_lst_detections_selected: number of detections selected is 4188 MCCs from 77075 to 77075
2018-12-13 08:03:52,847 - __main__ - DEBUG - Filtering loop: Number of detections for a LR mcc 77075 is 2
2018-12-13 08:03:52,847 - track_management - DEBUG - new_detections: Tested will be new 2 detections
2018-12-13 08:03:52,847 - track_management - DEBUG - new_detections: 	 	 with MCCs from 77075 to 77075
2018-12-13 08:03:52,847 - track_management - DEBUG - new_detections: In a _lst_not_assigned_detections is 3 detections.
2018-12-13 08:03:52,847 - track_management - DEBUG - new_detections: 	 after 10 mccs removal: 3 detections.
2018-12-13 08:03:52,847 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77075
2018-12-13 08:03:52,847 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77075
2018-12-13 08:03:52,847 - data_containers - DEBUG - 	 at x: 7.268175204039129, y: 3.1360386665247444
2018-12-13 08:03:52,847 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:52,848 - data_containers - DEBUG - 				a permutation of det1 77073, det2 77074
2018-12-13 08:03:52,848 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:52,848 - data_containers - DEBUG - 					 det2 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:52,848 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:52,848 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:52,848 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:52,848 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:52,848 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:52,848 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:52,848 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:52,848 - data_containers - DEBUG - 				a permutation of det1 77073, det2 77074
2018-12-13 08:03:52,848 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:52,849 - data_containers - DEBUG - 					 det2 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:52,849 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:52,849 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:52,849 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:52,849 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:52,849 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:52,849 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:52,849 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:52,850 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77073
2018-12-13 08:03:52,850 - data_containers - DEBUG - 					 det1 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:52,850 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:52,850 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:52,850 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:52,850 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:52,850 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:52,850 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:52,850 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:52,850 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:52,850 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77074
2018-12-13 08:03:52,850 - data_containers - DEBUG - 					 det1 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:52,850 - data_containers - DEBUG - 					 det2 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:52,851 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:52,851 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:52,851 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:52,851 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:52,851 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:52,851 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:52,851 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:52,851 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77073
2018-12-13 08:03:52,851 - data_containers - DEBUG - 					 det1 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:52,851 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:52,852 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:52,852 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:52,852 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:52,852 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:52,852 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:52,852 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:52,852 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:52,852 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77074
2018-12-13 08:03:52,852 - data_containers - DEBUG - 					 det1 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:52,852 - data_containers - DEBUG - 					 det2 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:52,853 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:52,853 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:52,853 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:52,853 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:52,853 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:52,853 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:52,853 - data_containers - INFO - UnAssignedDetectionList.new_detection: Detection stored in an unassigned list. Now it contains: 4 dets
2018-12-13 08:03:53,459 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77075
2018-12-13 08:03:53,460 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77075
2018-12-13 08:03:53,460 - data_containers - DEBUG - 	 at x: 9.966123186849071, y: 2.486225870304308
2018-12-13 08:03:53,460 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,461 - data_containers - DEBUG - 				a permutation of det1 77073, det2 77074
2018-12-13 08:03:53,461 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:53,461 - data_containers - DEBUG - 					 det2 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:53,462 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,462 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,462 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,462 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,463 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,463 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,463 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,463 - data_containers - DEBUG - 				a permutation of det1 77073, det2 77074
2018-12-13 08:03:53,464 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:53,464 - data_containers - DEBUG - 					 det2 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:53,464 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,464 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,464 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,464 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,465 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,465 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,465 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,465 - data_containers - DEBUG - 				a permutation of det1 77073, det2 77075
2018-12-13 08:03:53,465 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:53,465 - data_containers - DEBUG - 					 det2 x: 7.268175204039129, y: 3.1360386665247444
2018-12-13 08:03:53,465 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,465 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,466 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,466 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,466 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,466 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,466 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,466 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77073
2018-12-13 08:03:53,466 - data_containers - DEBUG - 					 det1 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:53,466 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:53,466 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,466 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,466 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,466 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,466 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,466 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,467 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,467 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77074
2018-12-13 08:03:53,467 - data_containers - DEBUG - 					 det1 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:53,467 - data_containers - DEBUG - 					 det2 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:53,467 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,467 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,467 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,467 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,467 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,467 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,467 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,467 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77075
2018-12-13 08:03:53,467 - data_containers - DEBUG - 					 det1 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:53,467 - data_containers - DEBUG - 					 det2 x: 7.268175204039129, y: 3.1360386665247444
2018-12-13 08:03:53,467 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,467 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,467 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,467 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point exists at: x=7.436610701325378, y=2.8779433926547258
2018-12-13 08:03:53,468 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,468 - data_containers - DEBUG - 	 Detection is in a gate with distance 0.
2018-12-13 08:03:53,468 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,468 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77073
2018-12-13 08:03:53,468 - data_containers - DEBUG - 					 det1 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:53,468 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:53,468 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,468 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,468 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,468 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,468 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,468 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,468 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,468 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77074
2018-12-13 08:03:53,468 - data_containers - DEBUG - 					 det1 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:53,468 - data_containers - DEBUG - 					 det2 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:53,468 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,468 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,468 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,468 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,468 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,468 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,468 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,468 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77075
2018-12-13 08:03:53,468 - data_containers - DEBUG - 					 det1 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:53,468 - data_containers - DEBUG - 					 det2 x: 7.268175204039129, y: 3.1360386665247444
2018-12-13 08:03:53,469 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,469 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,469 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,469 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,469 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,469 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,469 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,469 - data_containers - DEBUG - 				a permutation of det1 77075, det2 77073
2018-12-13 08:03:53,469 - data_containers - DEBUG - 					 det1 x: 7.268175204039129, y: 3.1360386665247444
2018-12-13 08:03:53,469 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:53,469 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,469 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,469 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,469 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,469 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,469 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,469 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,469 - data_containers - DEBUG - 				a permutation of det1 77075, det2 77074
2018-12-13 08:03:53,469 - data_containers - DEBUG - 					 det1 x: 7.268175204039129, y: 3.1360386665247444
2018-12-13 08:03:53,469 - data_containers - DEBUG - 					 det2 x: 7.09973970675288, y: 3.394133940394763
2018-12-13 08:03:53,469 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,469 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,469 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,470 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,470 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,470 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,470 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 4 
2018-12-13 08:03:53,470 - data_containers - DEBUG - 				a permutation of det1 77075, det2 77074
2018-12-13 08:03:53,470 - data_containers - DEBUG - 					 det1 x: 7.268175204039129, y: 3.1360386665247444
2018-12-13 08:03:53,470 - data_containers - DEBUG - 					 det2 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:53,470 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,470 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,470 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,470 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,470 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,470 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,470 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: searching for the best fit.
2018-12-13 08:03:53,470 - data_containers - DEBUG - 			There is 1 combinations where detection fit in a gate.
2018-12-13 08:03:53,470 - data_containers - DEBUG - 	 maximum distance is 0.
2018-12-13 08:03:53,470 - data_containers - DEBUG - Gate.__init__: diff_x = 2, diff_y = 2
2018-12-13 08:03:53,470 - data_containers - DEBUG - 			therefore a new track will be created.
2018-12-13 08:03:53,470 - data_containers - DEBUG - 			removed det 1: <data_containers.DetectionPoint object at 0x7f3c8a96bd68>.
2018-12-13 08:03:53,470 - data_containers - DEBUG - 			removed det 2: <data_containers.DetectionPoint object at 0x7f3c8a96bef0>.
2018-12-13 08:03:53,470 - data_containers - INFO - UnAssignedDetectionList.new_detection: Detection triggers a new track. 2 detections remain in a list of unassigned.
2018-12-13 08:03:53,979 - track_management - DEBUG - new_detections, no tracks: A new track was created. Currently 1 tracks is in the list.
2018-12-13 08:03:53,980 - data_containers - DEBUG - Track.init_tracker: Tracker initialized with: 
2018-12-13 08:03:53,980 - data_containers - DEBUG - 	 state vector x:	 7.09974
2018-12-13 08:03:53,981 - data_containers - DEBUG - 	 	 	 	 	 	 0.00226
2018-12-13 08:03:53,981 - data_containers - DEBUG - 	 	 	 	 	 	 3.39413
2018-12-13 08:03:53,981 - data_containers - DEBUG - 	 	 	 	 	 	 0.00108
2018-12-13 08:03:53,981 - track_management - DEBUG - new_detections, no tracks: tracker initialized for the new track: <tracking_filters.KalmanFilter object at 0x7f3c80ddd780>
2018-12-13 08:03:53,992 - data_containers - DEBUG - Track.start_tracker: Tracker started, current posteriori
2018-12-13 08:03:53,992 - data_containers - DEBUG -  		 x = 8.63003
2018-12-13 08:03:53,992 - data_containers - DEBUG -  		 y = 2.79730
2018-12-13 08:03:53,992 - track_management - DEBUG - new_detections, no tracks: new track's first 3 points: [<data_containers.TrackPoint object at 0x7f3c80e7d390>, <data_containers.TrackPoint object at 0x7f3c80e01518>, <data_containers.TrackPoint object at 0x7f3c80e01128>]
2018-12-13 08:03:53,992 - __main__ - DEBUG - Predict cycle for each track in a list of 1 started for LR at mcc: 77075.
2018-12-13 08:03:53,992 - data_containers - DEBUG - Track.predict: Tracker's predict cycle called, current apriori
2018-12-13 08:03:53,992 - data_containers - DEBUG -  		 x = 8.71364
2018-12-13 08:03:53,992 - data_containers - DEBUG -  		 y = 2.75847
2018-12-13 08:03:53,993 - __main__ - DEBUG - Type of ported new_track list <class 'data_containers.Track'>
2018-12-13 08:03:53,993 - __main__ - DEBUG - Type of ported new_track element <class 'data_containers.TrackPoint'>
2018-12-13 08:03:53,993 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:03:53,995 - data_containers - DEBUG - DetectionList.get_lst_detections_selected: number of detections selected is 4188 MCCs from 77076 to 77076
2018-12-13 08:03:53,995 - __main__ - DEBUG - Filtering loop: Number of detections for a LR mcc 77076 is 3
2018-12-13 08:03:53,995 - track_management - DEBUG - new_detections: Tested will be new 3 detections
2018-12-13 08:03:53,995 - track_management - DEBUG - new_detections: 	 	 with MCCs from 77076 to 77076
2018-12-13 08:03:53,995 - track_management - DEBUG - new_detections: In a _lst_not_assigned_detections is 2 detections.
2018-12-13 08:03:53,995 - track_management - DEBUG - new_detections: 	 after 10 mccs removal: 2 detections.
2018-12-13 08:03:53,995 - track_management - DEBUG - new_detections, tracks exist: Currently some tracks exist in a list. Will be scrutinized. Number of tracks: 1
2018-12-13 08:03:53,995 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:03:53,995 - track_management - DEBUG - new_detections, tracks exist: currently tested detection doesn't fit in.
2018-12-13 08:03:53,995 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77076
2018-12-13 08:03:53,996 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77076
2018-12-13 08:03:53,996 - data_containers - DEBUG - 	 at x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:53,996 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 2 
2018-12-13 08:03:53,996 - data_containers - DEBUG - 				a permutation of det1 77073, det2 77074
2018-12-13 08:03:53,996 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:53,996 - data_containers - DEBUG - 					 det2 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:53,996 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,996 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,996 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,996 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,996 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,996 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,996 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 2 
2018-12-13 08:03:53,996 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77073
2018-12-13 08:03:53,996 - data_containers - DEBUG - 					 det1 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:53,996 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:53,997 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:53,997 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:53,997 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:53,997 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:53,997 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:53,997 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:53,997 - data_containers - INFO - UnAssignedDetectionList.new_detection: Detection stored in an unassigned list. Now it contains: 3 dets
2018-12-13 08:03:54,395 - track_management - DEBUG - new_detections, tracks exist: Currently some tracks exist in a list. Will be scrutinized. Number of tracks: 1
2018-12-13 08:03:54,395 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:03:54,396 - track_management - DEBUG - new_detections, tracks exist: currently tested detection doesn't fit in.
2018-12-13 08:03:54,396 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77076
2018-12-13 08:03:54,396 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77076
2018-12-13 08:03:54,396 - data_containers - DEBUG - 	 at x: 7.305345867921791, y: 3.0443384369818016
2018-12-13 08:03:54,396 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:54,396 - data_containers - DEBUG - 				a permutation of det1 77073, det2 77074
2018-12-13 08:03:54,396 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:54,396 - data_containers - DEBUG - 					 det2 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:54,396 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:54,397 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:54,397 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:54,397 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:54,397 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:54,397 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:54,397 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:54,397 - data_containers - DEBUG - 				a permutation of det1 77073, det2 77076
2018-12-13 08:03:54,397 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:54,397 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:54,397 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:54,397 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:54,397 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:54,398 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point exists at: x=-6.610980701510598, y=43.12858360773065
2018-12-13 08:03:54,398 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:54,398 - data_containers - DEBUG - 	 Detection is in a gate with distance 0.
2018-12-13 08:03:54,398 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:54,398 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77073
2018-12-13 08:03:54,398 - data_containers - DEBUG - 					 det1 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:54,398 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:54,398 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:54,398 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:54,398 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:54,398 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:54,399 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:54,399 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:54,399 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:54,399 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77076
2018-12-13 08:03:54,399 - data_containers - DEBUG - 					 det1 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:54,399 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:54,399 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:54,399 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:54,399 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:54,399 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:54,399 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:54,399 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:54,399 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:54,399 - data_containers - DEBUG - 				a permutation of det1 77076, det2 77073
2018-12-13 08:03:54,399 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:54,400 - data_containers - DEBUG - 					 det2 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:54,400 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:54,400 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:54,400 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:54,400 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:54,400 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:54,400 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:54,400 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:03:54,400 - data_containers - DEBUG - 				a permutation of det1 77076, det2 77074
2018-12-13 08:03:54,400 - data_containers - DEBUG - 					 det1 x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:03:54,400 - data_containers - DEBUG - 					 det2 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:03:54,400 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:03:54,400 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:03:54,400 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:03:54,401 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:03:54,401 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:03:54,401 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:03:54,401 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: searching for the best fit.
2018-12-13 08:03:54,401 - data_containers - DEBUG - 			There is 1 combinations where detection fit in a gate.
2018-12-13 08:03:54,401 - data_containers - DEBUG - 	 maximum distance is 0.
2018-12-13 08:03:54,401 - data_containers - DEBUG - Gate.__init__: diff_x = 2, diff_y = 2
2018-12-13 08:03:54,401 - data_containers - DEBUG - 			therefore a new track will be created.
2018-12-13 08:03:54,401 - data_containers - DEBUG - 			removed det 1: <data_containers.DetectionPoint object at 0x7f3c8a96bba8>.
2018-12-13 08:03:54,401 - data_containers - DEBUG - 			removed det 2: <data_containers.DetectionPoint object at 0x7f3c8a96bcf8>.
2018-12-13 08:03:54,401 - data_containers - INFO - UnAssignedDetectionList.new_detection: Detection triggers a new track. 1 detections remain in a list of unassigned.
2018-12-13 08:04:07,588 - track_management - DEBUG - new_detections, no tracks: A new track was created. Currently 2 tracks is in the list.
2018-12-13 08:04:07,591 - data_containers - DEBUG - Track.init_tracker: Tracker initialized with: 
2018-12-13 08:04:07,591 - data_containers - DEBUG - 	 state vector x:	 -6.61098
2018-12-13 08:04:07,591 - data_containers - DEBUG - 	 	 	 	 	 	 3.33663
2018-12-13 08:04:07,591 - data_containers - DEBUG - 	 	 	 	 	 	 43.12858
2018-12-13 08:04:07,591 - data_containers - DEBUG - 	 	 	 	 	 	 -21.76741
2018-12-13 08:04:07,591 - track_management - DEBUG - new_detections, no tracks: tracker initialized for the new track: <tracking_filters.KalmanFilter object at 0x7f3c809b4cf8>
2018-12-13 08:04:07,592 - data_containers - DEBUG - Track.start_tracker: Tracker started, current posteriori
2018-12-13 08:04:07,592 - data_containers - DEBUG -  		 x = 0.51225
2018-12-13 08:04:07,592 - data_containers - DEBUG -  		 y = 21.56624
2018-12-13 08:04:07,592 - track_management - DEBUG - new_detections, no tracks: new track's first 3 points: [<data_containers.TrackPoint object at 0x7f3c80e62be0>, <data_containers.TrackPoint object at 0x7f3c80e62978>, <data_containers.TrackPoint object at 0x7f3c80e62780>]
2018-12-13 08:04:07,592 - track_management - DEBUG - new_detections, tracks exist: Currently some tracks exist in a list. Will be scrutinized. Number of tracks: 2
2018-12-13 08:04:07,592 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:07,592 - track_management - DEBUG - new_detections, tracks exist: none of tracks is active or they have been updated in this mcc
2018-12-13 08:04:07,592 - track_management - DEBUG - new_detections, tracks exist: currently tested detection doesn't fit in.
2018-12-13 08:04:07,592 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77076
2018-12-13 08:04:07,592 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77076
2018-12-13 08:04:07,593 - data_containers - DEBUG - 	 at x: 9.849461537346404, y: 3.1066202360473394
2018-12-13 08:04:07,593 - data_containers - INFO - UnAssignedDetectionList.new_detection:Detection stored in an unassigned list. Now it contains: 2 dets
2018-12-13 08:04:08,652 - __main__ - DEBUG - Predict cycle for each track in a list of 2 started for LR at mcc: 77076.
2018-12-13 08:04:08,653 - data_containers - DEBUG - Track.predict: Tracker's predict cycle called, current apriori
2018-12-13 08:04:08,653 - data_containers - DEBUG -  		 x = 8.79726
2018-12-13 08:04:08,653 - data_containers - DEBUG -  		 y = 2.71964
2018-12-13 08:04:08,653 - data_containers - DEBUG - Track.predict: Tracker's predict cycle called, current apriori
2018-12-13 08:04:08,653 - data_containers - DEBUG -  		 x = 1.10365
2018-12-13 08:04:08,653 - data_containers - DEBUG -  		 y = 18.16263
2018-12-13 08:04:08,653 - __main__ - DEBUG - Type of ported new_track list <class 'data_containers.Track'>
2018-12-13 08:04:08,653 - __main__ - DEBUG - Type of ported new_track element <class 'data_containers.TrackPoint'>
2018-12-13 08:04:08,654 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:04:08,655 - data_containers - DEBUG - DetectionList.get_lst_detections_selected: number of detections selected is 4188 MCCs from 77077 to 77077
2018-12-13 08:04:08,655 - __main__ - DEBUG - Filtering loop: Number of detections for a LR mcc 77077 is 2
2018-12-13 08:04:08,655 - track_management - DEBUG - new_detections: Tested will be new 2 detections
2018-12-13 08:04:08,655 - track_management - DEBUG - new_detections: 	 	 with MCCs from 77077 to 77077
2018-12-13 08:04:08,655 - track_management - DEBUG - new_detections: In a _lst_not_assigned_detections is 2 detections.
2018-12-13 08:04:08,656 - track_management - DEBUG - new_detections: 	 after 10 mccs removal: 2 detections.
2018-12-13 08:04:08,656 - track_management - DEBUG - new_detections, tracks exist: Currently some tracks exist in a list. Will be scrutinized. Number of tracks: 2
2018-12-13 08:04:08,656 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:08,656 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:08,656 - track_management - DEBUG - new_detections, tracks exist: currently tested detection doesn't fit in.
2018-12-13 08:04:08,656 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77077
2018-12-13 08:04:08,656 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77077
2018-12-13 08:04:08,656 - data_containers - DEBUG - 	 at x: 7.123170766149939, y: 3.337439871919813
2018-12-13 08:04:08,656 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 2 
2018-12-13 08:04:08,656 - data_containers - DEBUG - 				a permutation of det1 77074, det2 77076
2018-12-13 08:04:08,656 - data_containers - DEBUG - 					 det1 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:04:08,656 - data_containers - DEBUG - 					 det2 x: 9.849461537346404, y: 3.1066202360473394
2018-12-13 08:04:08,656 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:08,656 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:08,656 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:08,656 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point exists at: x=9.732664813227673, y=4.27627897144434
2018-12-13 08:04:08,656 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:08,656 - data_containers - DEBUG - 	 Detection is in a gate with distance 0.
2018-12-13 08:04:08,656 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 2 
2018-12-13 08:04:08,656 - data_containers - DEBUG - 				a permutation of det1 77076, det2 77074
2018-12-13 08:04:08,656 - data_containers - DEBUG - 					 det1 x: 9.849461537346404, y: 3.1066202360473394
2018-12-13 08:04:08,657 - data_containers - DEBUG - 					 det2 x: 9.966258261465134, y: 1.936961500650339
2018-12-13 08:04:08,657 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:08,657 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:08,657 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:08,657 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:04:08,657 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:08,657 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:04:08,657 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: searching for the best fit.
2018-12-13 08:04:08,657 - data_containers - DEBUG - 			There is 1 combinations where detection fit in a gate.
2018-12-13 08:04:08,657 - data_containers - DEBUG - 	 maximum distance is 0.
2018-12-13 08:04:08,657 - data_containers - DEBUG - Gate.__init__: diff_x = 2, diff_y = 2
2018-12-13 08:04:08,657 - data_containers - DEBUG - 			therefore a new track will be created.
2018-12-13 08:04:08,657 - data_containers - DEBUG - 			removed det 1: <data_containers.DetectionPoint object at 0x7f3c8a96beb8>.
2018-12-13 08:04:08,657 - data_containers - DEBUG - 			removed det 2: <data_containers.DetectionPoint object at 0x7f3c8a96bf60>.
2018-12-13 08:04:08,657 - data_containers - INFO - UnAssignedDetectionList.new_detection: Detection triggers a new track. 0 detections remain in a list of unassigned.
2018-12-13 08:04:37,997 - track_management - DEBUG - new_detections, no tracks: A new track was created. Currently 3 tracks is in the list.
2018-12-13 08:04:37,999 - data_containers - DEBUG - Track.init_tracker: Tracker initialized with: 
2018-12-13 08:04:37,999 - data_containers - DEBUG - 	 state vector x:	 9.96626
2018-12-13 08:04:37,999 - data_containers - DEBUG - 	 	 	 	 	 	 0.00908
2018-12-13 08:04:37,999 - data_containers - DEBUG - 	 	 	 	 	 	 1.93696
2018-12-13 08:04:37,999 - data_containers - DEBUG - 	 	 	 	 	 	 0.00177
2018-12-13 08:04:38,000 - track_management - DEBUG - new_detections, no tracks: tracker initialized for the new track: <tracking_filters.KalmanFilter object at 0x7f3c80c17160>
2018-12-13 08:04:38,001 - data_containers - DEBUG - Track.start_tracker: Tracker started, current posteriori
2018-12-13 08:04:38,001 - data_containers - DEBUG -  		 x = 8.47257
2018-12-13 08:04:38,001 - data_containers - DEBUG -  		 y = 3.21696
2018-12-13 08:04:38,001 - track_management - DEBUG - new_detections, no tracks: new track's first 3 points: [<data_containers.TrackPoint object at 0x7f3c80caf748>, <data_containers.TrackPoint object at 0x7f3c80caf160>, <data_containers.TrackPoint object at 0x7f3c80caf588>]
2018-12-13 08:04:38,001 - track_management - DEBUG - new_detections, tracks exist: Currently some tracks exist in a list. Will be scrutinized. Number of tracks: 3
2018-12-13 08:04:38,001 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:38,001 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:38,001 - track_management - DEBUG - new_detections, tracks exist: none of tracks is active or they have been updated in this mcc
2018-12-13 08:04:38,002 - track_management - DEBUG - new_detections, tracks exist: currently tested detection doesn't fit in.
2018-12-13 08:04:38,002 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77077
2018-12-13 08:04:38,002 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77077
2018-12-13 08:04:38,002 - data_containers - DEBUG - 	 at x: 9.930809910676054, y: 1.5778470338155324
2018-12-13 08:04:38,002 - data_containers - INFO - UnAssignedDetectionList.new_detection:Detection stored in an unassigned list. Now it contains: 1 dets
2018-12-13 08:04:41,019 - __main__ - DEBUG - Predict cycle for each track in a list of 3 started for LR at mcc: 77077.
2018-12-13 08:04:41,019 - data_containers - DEBUG - Track.predict: Tracker's predict cycle called, current apriori
2018-12-13 08:04:41,019 - data_containers - DEBUG -  		 x = 8.88088
2018-12-13 08:04:41,019 - data_containers - DEBUG -  		 y = 2.68081
2018-12-13 08:04:41,019 - data_containers - DEBUG - Track.predict: Tracker's predict cycle called, current apriori
2018-12-13 08:04:41,019 - data_containers - DEBUG -  		 x = 1.69505
2018-12-13 08:04:41,019 - data_containers - DEBUG -  		 y = 14.75902
2018-12-13 08:04:41,019 - data_containers - DEBUG - Track.predict: Tracker's predict cycle called, current apriori
2018-12-13 08:04:41,019 - data_containers - DEBUG -  		 x = 8.38869
2018-12-13 08:04:41,020 - data_containers - DEBUG -  		 y = 3.23210
2018-12-13 08:04:41,020 - __main__ - DEBUG - Type of ported new_track list <class 'data_containers.Track'>
2018-12-13 08:04:41,020 - __main__ - DEBUG - Type of ported new_track element <class 'data_containers.TrackPoint'>
2018-12-13 08:04:41,020 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:04:41,021 - data_containers - DEBUG - DetectionList.get_lst_detections_selected: number of detections selected is 4188 MCCs from 77078 to 77078
2018-12-13 08:04:41,021 - __main__ - DEBUG - Filtering loop: Number of detections for a LR mcc 77078 is 2
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections: Tested will be new 2 detections
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections: 	 	 with MCCs from 77078 to 77078
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections: In a _lst_not_assigned_detections is 1 detections.
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections: 	 after 10 mccs removal: 1 detections.
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections, tracks exist: Currently some tracks exist in a list. Will be scrutinized. Number of tracks: 3
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections, tracks exist: currently tested detection doesn't fit in.
2018-12-13 08:04:41,021 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77078
2018-12-13 08:04:41,021 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77078
2018-12-13 08:04:41,021 - data_containers - DEBUG - 	 at x: 7.2499943085415515, y: 3.095045397471739
2018-12-13 08:04:41,022 - data_containers - INFO - UnAssignedDetectionList.new_detection:Detection stored in an unassigned list. Now it contains: 2 dets
2018-12-13 08:04:42,382 - track_management - DEBUG - new_detections, tracks exist: Currently some tracks exist in a list. Will be scrutinized. Number of tracks: 3
2018-12-13 08:04:42,383 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:42,384 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:42,384 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:42,384 - track_management - DEBUG - new_detections, tracks exist: currently tested detection doesn't fit in.
2018-12-13 08:04:42,384 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77078
2018-12-13 08:04:42,384 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77078
2018-12-13 08:04:42,384 - data_containers - DEBUG - 	 at x: 9.959080895355383, y: 2.420779821240279
2018-12-13 08:04:42,384 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 2 
2018-12-13 08:04:42,384 - data_containers - DEBUG - 				a permutation of det1 77077, det2 77078
2018-12-13 08:04:42,384 - data_containers - DEBUG - 					 det1 x: 9.930809910676054, y: 1.5778470338155324
2018-12-13 08:04:42,384 - data_containers - DEBUG - 					 det2 x: 7.2499943085415515, y: 3.095045397471739
2018-12-13 08:04:42,385 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:42,385 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:42,385 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:42,385 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:04:42,385 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:42,385 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:04:42,385 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 2 
2018-12-13 08:04:42,385 - data_containers - DEBUG - 				a permutation of det1 77078, det2 77077
2018-12-13 08:04:42,385 - data_containers - DEBUG - 					 det1 x: 7.2499943085415515, y: 3.095045397471739
2018-12-13 08:04:42,385 - data_containers - DEBUG - 					 det2 x: 9.930809910676054, y: 1.5778470338155324
2018-12-13 08:04:42,385 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:42,385 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:42,385 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:42,385 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:04:42,385 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:42,386 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:04:42,386 - data_containers - INFO - UnAssignedDetectionList.new_detection: Detection stored in an unassigned list. Now it contains: 3 dets
2018-12-13 08:04:43,218 - __main__ - DEBUG - Predict cycle for each track in a list of 3 started for LR at mcc: 77078.
2018-12-13 08:04:43,220 - data_containers - DEBUG - Track.predict: Tracker's predict cycle called, current apriori
2018-12-13 08:04:43,220 - data_containers - DEBUG -  		 x = 8.96450
2018-12-13 08:04:43,220 - data_containers - DEBUG -  		 y = 2.64198
2018-12-13 08:04:43,221 - data_containers - DEBUG - Track.predict: Tracker's predict cycle called, current apriori
2018-12-13 08:04:43,221 - data_containers - DEBUG -  		 x = 2.28645
2018-12-13 08:04:43,221 - data_containers - DEBUG -  		 y = 11.35542
2018-12-13 08:04:43,221 - data_containers - DEBUG - Track.predict: Tracker's predict cycle called, current apriori
2018-12-13 08:04:43,221 - data_containers - DEBUG -  		 x = 8.30480
2018-12-13 08:04:43,221 - data_containers - DEBUG -  		 y = 3.24725
2018-12-13 08:04:43,221 - __main__ - DEBUG - Type of ported new_track list <class 'data_containers.Track'>
2018-12-13 08:04:43,221 - __main__ - DEBUG - Type of ported new_track element <class 'data_containers.TrackPoint'>
2018-12-13 08:04:43,221 - data_containers - DEBUG - DetectionList.__init__: list initialized
2018-12-13 08:04:43,224 - data_containers - DEBUG - DetectionList.get_lst_detections_selected: number of detections selected is 4188 MCCs from 77079 to 77079
2018-12-13 08:04:43,224 - __main__ - DEBUG - Filtering loop: Number of detections for a LR mcc 77079 is 3
2018-12-13 08:04:43,224 - track_management - DEBUG - new_detections: Tested will be new 3 detections
2018-12-13 08:04:43,224 - track_management - DEBUG - new_detections: 	 	 with MCCs from 77079 to 77079
2018-12-13 08:04:43,224 - track_management - DEBUG - new_detections: In a _lst_not_assigned_detections is 3 detections.
2018-12-13 08:04:43,224 - track_management - DEBUG - new_detections: 	 after 10 mccs removal: 3 detections.
2018-12-13 08:04:43,224 - track_management - DEBUG - new_detections, tracks exist: Currently some tracks exist in a list. Will be scrutinized. Number of tracks: 3
2018-12-13 08:04:43,225 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:43,225 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:43,225 - track_management - DEBUG - new_detections, tracks exist: The vector of all distances from each track's gate center, the aim, is: 0.000
2018-12-13 08:04:43,225 - track_management - DEBUG - new_detections, tracks exist: currently tested detection doesn't fit in.
2018-12-13 08:04:43,225 - track_management - DEBUG - new_detections, no track exists yet. Processing detection at mcc: 77079
2018-12-13 08:04:43,225 - data_containers - INFO - UnAssignedDetectionList.new_detection: tested new detection with MCC: 77079
2018-12-13 08:04:43,225 - data_containers - DEBUG - 	 at x: -6.610980701510598, y: 43.12858360773065
2018-12-13 08:04:43,225 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:04:43,225 - data_containers - DEBUG - 				a permutation of det1 77077, det2 77078
2018-12-13 08:04:43,225 - data_containers - DEBUG - 					 det1 x: 9.930809910676054, y: 1.5778470338155324
2018-12-13 08:04:43,225 - data_containers - DEBUG - 					 det2 x: 7.2499943085415515, y: 3.095045397471739
2018-12-13 08:04:43,225 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:43,226 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:43,226 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:43,226 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:04:43,226 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:43,226 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:04:43,226 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:04:43,226 - data_containers - DEBUG - 				a permutation of det1 77077, det2 77078
2018-12-13 08:04:43,226 - data_containers - DEBUG - 					 det1 x: 9.930809910676054, y: 1.5778470338155324
2018-12-13 08:04:43,226 - data_containers - DEBUG - 					 det2 x: 9.959080895355383, y: 2.420779821240279
2018-12-13 08:04:43,226 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:43,226 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:43,226 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:43,227 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point exists at: x=9.987351880034712, y=3.2637126086650254
2018-12-13 08:04:43,227 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:43,227 - data_containers - DEBUG - 	 Detection is in a gate with distance 0.
2018-12-13 08:04:43,227 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:04:43,227 - data_containers - DEBUG - 				a permutation of det1 77078, det2 77077
2018-12-13 08:04:43,227 - data_containers - DEBUG - 					 det1 x: 7.2499943085415515, y: 3.095045397471739
2018-12-13 08:04:43,227 - data_containers - DEBUG - 					 det2 x: 9.930809910676054, y: 1.5778470338155324
2018-12-13 08:04:43,227 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:43,227 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:43,227 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:43,227 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:04:43,227 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:43,227 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:04:43,228 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:04:43,228 - data_containers - DEBUG - 				a permutation of det1 77078, det2 77078
2018-12-13 08:04:43,228 - data_containers - DEBUG - 					 det1 x: 7.2499943085415515, y: 3.095045397471739
2018-12-13 08:04:43,228 - data_containers - DEBUG - 					 det2 x: 9.959080895355383, y: 2.420779821240279
2018-12-13 08:04:43,228 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:43,228 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:43,228 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:43,228 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:04:43,228 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:43,228 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:04:43,228 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:04:43,228 - data_containers - DEBUG - 				a permutation of det1 77078, det2 77077
2018-12-13 08:04:43,228 - data_containers - DEBUG - 					 det1 x: 9.959080895355383, y: 2.420779821240279
2018-12-13 08:04:43,228 - data_containers - DEBUG - 					 det2 x: 9.930809910676054, y: 1.5778470338155324
2018-12-13 08:04:43,228 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:43,229 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:43,229 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:43,229 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:04:43,229 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:43,229 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:04:43,229 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: number of unassigned detections in a list: 3 
2018-12-13 08:04:43,229 - data_containers - DEBUG - 				a permutation of det1 77078, det2 77078
2018-12-13 08:04:43,229 - data_containers - DEBUG - 					 det1 x: 9.959080895355383, y: 2.420779821240279
2018-12-13 08:04:43,229 - data_containers - DEBUG - 					 det2 x: 7.2499943085415515, y: 3.095045397471739
2018-12-13 08:04:43,229 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of azimuths is not defined, a criteria az is always True
2018-12-13 08:04:43,229 - data_containers - DEBUG - DetectionPoint.test_in_range_of: extent of beams is not defined, a criteria beam is always True
2018-12-13 08:04:43,229 - data_containers - DEBUG - DetectionPoint.test_in_range_of: tested detection is older than a self
2018-12-13 08:04:43,230 - data_containers - DEBUG - UnAssignedDetectionList.two_point_projection: projected point does not exist, det2 not in range of det1.
2018-12-13 08:04:43,230 - data_containers - DEBUG - Gate.__init__: diff_x = 0.5, diff_y = 0.3
2018-12-13 08:04:43,230 - data_containers - DEBUG - 	 Detection doesn't fit within the gate.
2018-12-13 08:04:43,230 - data_containers - DEBUG - UnAssignedDetectionList.new_detection: searching for the best fit.
2018-12-13 08:04:43,230 - data_containers - DEBUG - 			There is 1 combinations where detection fit in a gate.
2018-12-13 08:04:43,230 - data_containers - DEBUG - 	 maximum distance is 0.
2018-12-13 08:04:43,230 - data_containers - DEBUG - Gate.__init__: diff_x = 2, diff_y = 2
2018-12-13 08:04:43,230 - data_containers - DEBUG - 			therefore a new track will be created.
2018-12-13 08:04:43,230 - data_containers - DEBUG - 			removed det 1: <data_containers.DetectionPoint object at 0x7f3c8a96bf28>.
2018-12-13 08:04:43,230 - data_containers - DEBUG - 			removed det 2: <data_containers.DetectionPoint object at 0x7f3c8a96bb38>.
2018-12-13 08:04:43,230 - data_containers - INFO - UnAssignedDetectionList.new_detection: Detection triggers a new track. 1 detections remain in a list of unassigned.
